### PR TITLE
Refactor service changes 

### PR DESCRIPTION
**Pull request by Jeyaprakash-NK**
1. Service files refactor changes
2. MUI  css animation changes
3. Bug fixes
4. Logo changes

### CODE REVIEW

File: src/cluster/clusterServices.tsx [hunk 1]
```diff
@@ -0,0 +1,360 @@
...
+  clusterName: string;
+  clusterUuid: string;
+  projectId?: string;
+  regionId?: string;
+}
+export class ClusterService {
+  static listClustersAPIService = async (
+    setProjectId: (value: string) => void,
+    renderActions: (value: IClusterRenderData) => React.JSX.Element,
+    setclustersList: (value: any) => void,
```
Comment (ptwng) [inline src/cluster/clusterServices.tsx:60]: type please!

Comment (ptwng) [reply src/cluster/clusterServices.tsx:60]: also setClustersList

File: src/cluster/clusterServices.tsx [hunk 2]
```diff
@@ -0,0 +1,360 @@
...
+      toast.error('Failed to fetch clusters', toastifyCustomStyle);
+    }
+  };
+
+  static getClusterDetailsService = async (
+    setProjectName: (value: string) => void,
+    clusterSelected: string,
+    setErrorView: (value: boolean) => void,
+    setIsLoading: (value: boolean) => void,
+    setClusterInfo: any
```
Comment (ptwng) [inline src/cluster/clusterServices.tsx:146]: ditto

File: src/cluster/clusterServices.tsx [hunk 3]
```diff
@@ -0,0 +1,360 @@
...
+    try {
+      const response = await authenticatedFetch({
+        uri: `clusters/${selectedCluster}:stop`,
+        method: HTTP_METHOD.POST,
+        regionIdentifier: 'regions'
+      });
+      const formattedResponse = await response.json();
+      console.log(formattedResponse);
+      listClustersAPI();
+      timer.current = setInterval(() => {
```
Comment (ptwng) [inline src/cluster/clusterServices.tsx:247]: when does this timer get cleared?

File: src/controls/MuiWrappedInput.tsx [hunk 4]
```diff
@@ -14,25 +14,24 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { TextField, styled } from '@mui/material';
+import {  TextField, styled } from '@mui/material';
```
Comment (ptwng) [inline src/controls/MuiWrappedInput.tsx:17]: nit: some odd whitespace formatting here and below

**Review by ptwng — CHANGES_REQUESTED** [review]

File: src/cluster/clusterServices.tsx (hunk 1 above)
Comment (Jeyaprakash-NK) [reply src/cluster/clusterServices.tsx]: Resolved: type added and setClustersList changed

File: src/cluster/clusterServices.tsx (hunk 2 above)
Comment (Jeyaprakash-NK) [reply src/cluster/clusterServices.tsx]: Resolved: type added

File: src/cluster/clusterServices.tsx (hunk 3 above)
Comment (Jeyaprakash-NK) [reply src/cluster/clusterServices.tsx:259]: Cleared in statusApiService After cluster status changed to 'STOP' clearInterval executed

File: src/controls/MuiWrappedInput.tsx (hunk 4 above)
Comment (Jeyaprakash-NK) [reply src/controls/MuiWrappedInput.tsx]: Resolved: Handled Prettier in all files

**Review by ptwng — APPROVED** [review]

**Pull request merged**